x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -27,7 +27,7 @@ import type { SecurityTelemetryTaskConfig } from './task';
 import { SecurityTelemetryTask } from './task';
 import { telemetryConfiguration } from './configuration';
 
-const usageLabelPrefix: string[] = ['security_telemetry', 'sender'];
+export const usageLabelPrefix: string[] = ['security_telemetry', 'sender'];
```

Comment:
On second look, you don't need to export this as I believe the original author of the code intended it to be used only in the `sender.ts` class.

I think you just want to have a local variable in the tasks themselves like

```ts
const usageLabelPrefix: string[] = ['security_telemetry', 'lists'];
```